docs/custom-flows/bot-sign-up-protection.mdx

```diff
@@ -1,64 +1,165 @@
 </Callout>
 
+Clerk provides the ability to add a CAPTCHA widget to your sign-up flows to protect against bot sign-ups. This guide will show you how to add the CAPTCHA widget to your custom sign-up flow.
 
+<Steps>
 
-If you have enabled the Smart Bot Protection widget under **User & Authentication > Attack Protection > Bot sign-up protection** in Dashboard, it's required to include a specific element in your DOM, to enable rendering of the widget.
+### Enable bot sign-up protection
 
-Specifically, there should be a `<div id="clerk-captcha" />` element by the time you call `signUp.create()`. This element acts as a placeholder onto which the widget will be rendered.
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection).
+1. Go to **User & Authentication > Attack Protection** in the sidebar menu.
+1. In the **Bot sign-up protection** section, enable the feature and choose the **CAPTCHA type** you want to use.
 
-If this element is not found, the SDK will transparently fall back to the Invisible widget, in order to avoid breaking your sign-up flow. If this happens, you should see a relevant error in your browser's console.
+### Add the CAPTCHA widget to your custom sign-up form
 
-The Invisible widget is more prone to false positives, therefore we strongly recommend that you fix this error by ensuring the `<div id="clerk-captcha" />` element exists in your DOM.
+To render the CAPTCHA widget in your custom sign-up form, you need to include a specific element in your DOM. Specifically, there should be a `<div id="clerk-captcha" />` element by the time you call `signUp.create()`. This element acts as a placeholder onto which the widget will be rendered.
 
-## Example
-```jsx
-import { useState } from "react";
-import { useSignUp } from "@clerk/nextjs";
- 
-export default function SignUpForm() {
-  const { isLoaded, signUp, setActive } = useSignUp();
-  const [emailAddress, setEmailAddress] = useState("");
-  const [password, setPassword] = useState("");
+If this element is not found, the SDK will transparently fall back to the **Invisible** widget in order to avoid breaking your sign-up flow. If this happens, you should see a relevant error in your browser's console.
 
-  const handleSubmit = async (e) => {
-    e.preventDefault();
-    try {
-      await signUp.create({
-        emailAddress,
-        password,
-      });
-    } catch (err: any) {
-      console.error('sign-up error');
-    }
-  };
- 
-  return (
-    <div>
-        <form>
+> [!TIP]
+> The **Invisible** widget automatically blocks suspected bot traffic without user intervention. Therefore, it's strongly recommended that you ensure the `<div id="clerk-captcha" />` element exists in your DOM.
```

Comment:
```suggestion
> The **Invisible** widget automatically blocks suspected bot traffic without offering users falsely detected as bots with an opportunity to prove otherwise. Therefore, it's strongly recommended that you ensure the `<div id="clerk-captcha" />` element exists in your DOM.
```